packages/migrate/migrations/svelte-5/index.js

```diff
@@ -56,6 +58,42 @@ export async function migrate() {
 		}
 	}
 
+	const kit_dep = pkg.devDependencies?.['@sveltejs/kit'] ?? pkg.dependencies?.['@sveltejs/kit'];
+	if (kit_dep && semver.validRange(kit_dep) && semver.gtr('2.0.0', kit_dep)) {
+		console.log(
+			colors
+				.bold()
+				.yellow(
+					'\nDetected SvelteKit 1. You need to upgrade to SvelteKit version 2 first (`npx svelte-migrate sveltekit-2`).\n'
+				)
+		);
+		const response = await prompts({
+			type: 'confirm',
+			name: 'value',
+			message: 'Run `sveltekit-2` migration now?',
```

Comment:
I'm not sure it makes sense to print backticks to the terminal?
```suggestion
			message: 'Run sveltekit-2 migration now?',
```